### ISSUE TITLE
Preserve the size of "Open Database Object" window

### ISSUE THREAD

**Issue by informatik01** [dbeaver/dbeaver#23244]
Hi.

It would be really convenient if the developers would implement functionality for preserving the size, especially the width, of the "Open Database Objects..." window (top menu "Navigate > Open Database Objects..."). When I open it, it is always has the same size even if I previously changed it (made wider):
<img width="380" alt="image" src="https://github.com/dbeaver/dbeaver/assets/11339880/14b75049-d1c0-4b74-a588-3286ce25c478">

I was expecting that if I change the size / width of this window, next time when I open it - it will be of the same width. Most of the time search results do not fully fit the window width (because tables are shown with the related fully qualified names / schemas), that is why I almost always have to increase the width of this window to have an _aesthetically pleasant_ view of the results.

So it would be great to have the above described feature in future versions of DBeaver.

Thank you.

---
_This feature request was initially created as part of DBeaver "Discussion":_
https://github.com/orgs/dbeaver/discussions/23213#discussion-6410755

**Comment by LonwoLonwo** [comment]
Thanks for the idea @informatik01

**Comment by informatik01** [comment]
### UPDATE

Just noticed that the requested functionality perfectly works in cases when you try to open a database object AND no connection is selected: in such cases a pop-up window appears notifying a user that no connection is selected ("No active datasource") and this window nicely preserves the last selected size (both width and height) and even _position_ on a screen.

<img width="467" alt="image" src="https://github.com/dbeaver/dbeaver/assets/11339880/07f80c04-c11d-474f-be25-361fcc93fccb">

So in other words, the requested functionality is already present (_indirectly_), but ONLY for the notification window.
FYI.
